json/request.json

```diff
@@ -0,0 +1,12 @@
+{
+  "request list_caches": true,
+  "request statistics": true,
+  "statistics request info": {
+    "cache_id": 1,
+    "core_id": 0,
+    "io_class": 0,
```

Comment:
It would make sense to make "core_id" and "io_class" parameters optional

Reply:
Problem addressed trough increase granulation and making current parameters obligatory (different commands for each statistics level)